front/app/containers/App/index.tsx

```diff
@@ -417,10 +404,30 @@ const App = ({ children }: Props) => {
                   <ConsentManager />
                 </Suspense>
               </ErrorBoundary>
-              <ErrorBoundary>
-                <MainHeader setRef={setNavbarRef} />
-              </ErrorBoundary>
-              <InnerContainer>
+              {showFrontOfficeNavbar && (
+                <ErrorBoundary>
+                  <MainHeader setRef={setNavbarRef} />
+                </ErrorBoundary>
+              )}
+              <Box
+                width="100vw"
+                display="flex"
+                flexDirection="column"
+                alignItems="stretch"
+                pt={
+                  showFrontOfficeNavbar
+                    ? `${stylingConsts.menuHeight}px`
+                    : undefined
+                }
+                minHeight={
+                  isSmallerThanTablet
+                    ? `calc(100vh - ${
```

Comment:
Previously, if `menuHeight` was 10 and `mobileMenuHeight` was 5, the result would be `100vh - 15px`. And now it'll be `100vh - 5px`. Is it expected?

Reply:
Using the same calculations that were there. I have just moved them to use a Box component

Reply:
It doesn't seem the same to me (maybe I'm wrong). Now, the subtraction is in brackets `100 - (height - mobileHeight)` which is the same as `100 - height + mobileHeight`. And previously, it was two minus signs.

Reply:
Let me double check. That might be an error

Reply:
Fixed [here](https://github.com/CitizenLabDotCo/citizenlab/pull/4619)